Docker/build/WDQS/runBlazegraph.sh

```diff
@@ -2,5 +2,6 @@
 # This file is provided by the wikibase/wdqs docker image.
 
 cd /wdqs || exit
+shift
```

Comment:
I'm confused by why the shift is here?

```
$ ./runBlazegraph.sh arg1 arg2 arg3
./runBlazegraph.sh arg2 arg3
```

Why do we not want the fist argument to be passe din?

Reply:
Yes, this is wrong. I was under the assumption that `$@ == [$0:$N]` but it is indeed `[$1:$N]`.